redwood-protocol-widget/src/commonMain/kotlin/app/cash/redwood/protocol/widget/DiffConsumingNode.kt

```diff
@@ -16,17 +16,29 @@
 package app.cash.redwood.protocol.widget
 
 import app.cash.redwood.protocol.EventSink
+import app.cash.redwood.protocol.Id
 import app.cash.redwood.protocol.PropertyDiff
 import app.cash.redwood.widget.Widget
 import kotlinx.serialization.json.JsonArray
 
 /**
  * A [Widget] which consumes protocol diffs and applies them to a platform-specific representation.
+ *
+ * @suppress
  */
-public interface DiffConsumingWidget<W : Any> : Widget<W> {
-  public fun apply(diff: PropertyDiff, eventSink: EventSink)
+public abstract class DiffConsumingNode<W : Any>(
```

Comment:
This type will eventually be annotated with `@RedwoodCodegenApi` but that currently also affects the nested `Factory` which is public API. These two types will split in a follow-up at which point this class will receive the annotation.